ros/api/v1/recommendations.py

```diff
@@ -31,18 +31,19 @@ class RecommendationsApi(Resource):
 
     @marshal_with(data_fields)
     def get(self, host_id):
+        system = ''
```

Comment:
ditto
```suggestion
        system = None
```
`system` variable holds an object so does it make sense to initialize it to `None`?